docs/DeveloperGuide.md

```diff
@@ -523,3 +703,32 @@ testers are expected to do more *exploratory* testing.
 
    4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
+
+---
+
+## **Appendix: Planned Enhancements**
+
+1. **Command Consistency Improvements:**
+    * Standardize the behavior of the `delete` command to reset the person list post-execution, aligning with the current behavior of `add` and `edit` commands. This change will improve user experience by providing consistent feedback and list visibility.
+2. **Email Validation Enhancement:**
+    * Refine the email validity checks to include verification of the domain, ensuring that the email not only contains an '@' symbol followed by any alphanumeric characters, but also follows a valid format with a recognized domain. This enhancement aims to reduce incorrect/invalid email entries and improve data integrity.
+3. **Email Search Logic Enhancement:**
+    * Revamp the find functionality for the email field to accommodate searches without the typical formatting of a valid email address, thus broadening the search capabilities.
+4. **Duplicate Handling Improvements:**
+    * Revamp the duplicate detection logic to identify duplicates based on phone numbers and email addresses, ignoring whitespace variations. 
+    * Names will no longer be considered for duplicate checks due to the possibility of multiple individuals sharing the same name, thus mimicking real-world scenarios more accurately.
+5. **Cross-Platform UI Consistency:**
+6. **Detail Panel Behavior Correction:**
+    * Adjust the behavior of the Detail Panel to clear its contents when the last person in the person list is deleted, ensuring that no outdated information remains visible, thereby preventing user confusion.
+    * Implement font packaging within the software to ensure UI consistency across different operating systems, particularly addressing layout issues on MacOS caused by missing font styles.
```

Comment:
I think this line belongs to point 5. above, which is missing explanations.

Reply:
Ah yes you're right